backend/tests/test_project.py

```diff
@@ -1,7 +1,7 @@
 import json
 import unittest
 import uuid
-from flask import current_app
+from flask import current_app, request
```

Comment:
@peterghrong Is there something wrong with your editor? This keeps on showing up (unused import request).

Reply:
Removed